### ISSUE TITLE
Cliente se Desconecta y Conecta 

### ISSUE THREAD

**Issue by JVeloz19** [IIC2233/Syllabus#614]
### Prerrequisitos
(Marcar colocando una `X` entre los corchetes los ítems que ya hiciste, así: `"[X]"`)

* [X] Leí las reglas del foro (https://github.com/IIC2233/Syllabus/wiki/3.-Reglas-del-foro-(issues)).
* [X] Revisé el compilado de dudas de la tarea y no encontré una *issue* similar a la mía (https://github.com/IIC2233/Syllabus/issues/421).
* [X] Mi duda no se trata sobre una librería o *built-in*, ya que eso se pregunta en la *issue* creada para ello ([click aquí para ir a esa _issue_](/IIC2233/Syllabus/issues/420)).
* [X] De ser necesario, colocaré código simple que permita explicar mi problema o duda, ya que compartir código de esta evaluación puede ser considerado una falta a la ética.

### Duda

Hola! 
En el enunciado se habla muy poco de lo que ocurre cuando un usuario se desconecta, solo se sabe que primero no pueden haber 2 personas conectadas a la vez con el mismo username, y además cuando se desconecta y reconecta puede perder algún archivo por el evento. 

Sin embargo me surgen una duda: Que pasa si cuando se desconecta un user y  luego entra con otro username? Usara el mismo directorio de descargas que tenia anteriormente o se considera un cliente nuevo? O bien, no pasa nada si otro user entra con el username de alguien que esta desconectado?

**Comment by PeterMassmann** [comment]
Efectivamente, cada username debe tener una carpeta de descargas distinta. Por lo tanto, si te loggeas con un username distinto, tendrás archivos descargando distintos, actuando como clientes completamente diferentes.

**Comment by JVeloz19** [comment]
Oka, pero me sigue surgiendo la duda de si me loggeo en otra ventana con el username de alguien que estaba antes y tenia sus descargas, como que no me hace sentido que por ejemplo me loggeo en la cuenta de alguien desde otro computador tenga los archivos que tenia en el otro pc. O simplemente ignoro esta intuicion?

**Comment by MTGG123** [comment]
> Efectivamente, cada username debe tener una carpeta de descargas distinta. Por lo tanto, si te loggeas con un username distinto, tendrás archivos descargando distintos, actuando como clientes completamente diferentes.

Holaaa, yo también tengo dudas con esto ¿Cómo podríamos iniciar sesión con el nombre de otro usuario si al inicio nos piden que verifiquemos que el nombre usuario no existe dentro del listado de clientes del servidor? También, el enunciado menciona que el registro de archivos de un cliente que tiene el servidor no es confiable y que debemos verificar si el archivo no fue eliminado, pero ¿Cómo podemos recuperar los archivos de un usuario que se reconecta si no es por medio del registro que tiene el servidor? Gracias :)
